src/HtmlUtils.js

```diff
@@ -44,8 +45,16 @@ var sanitizeHtmlParams = {
     allowedSchemesByTag: {},
     
     transformTags: { // custom to matrix
-        // add blank targets to all hyperlinks
-        'a': sanitizeHtml.simpleTransform('a', { target: '_blank'} )
+        // add blank targets to all hyperlinks except vector URLs
+        'a': function(tagName, attribs) {
+            var m = attribs.href ? attribs.href.match(linkifyMatrix.VECTOR_URL_PATTERN) : null;
+            if (m) {
+                return { tagName: 'a', attribs: { href: attribs.href, name: attribs.name } };
+            }
+            else {
+                return { tagName: 'a', attribs: { href: attribs.href, name: attribs.name, target: '_blank' } };
```

Comment:
might be preferable to modify `attribs['target']` in-place (as `simpleTransform` does) rather than hardcode a list of attributes which we preserve? Can argue this both ways though.